client/jetpack-connect/style.scss

```diff
@@ -936,6 +936,13 @@ $colophon-height: 50px; // wpcomColophon element at the bottom
 		background-color: var( --color-white );
 		border-radius: 3px;
 	}
+
+	.site-verticals-suggestion-search__topic-list-item {
+		
```

Comment:
Nit: I usually refrain from newlines in the beginning/end of a nested block.

Reply:
Oh, I _always_ do it in SCSS so it's clear they don't have any direct styles applied to them. 🙃

Reply:
I'm happy to change this and it shouldn't hold us back. Do you have any strong opinions on either, @shaunandrews ?

Reply:
Same as @tyxla, I never do the empty line — its worth noting you can remove the nesting here altogether by just adding the focus selector to the parent selector. `.site-verticals-suggestion-search__topic-list-item:focus`

Reply:
Anyway, this is really too minor to hold this PR - feel free to ship as-is.

Reply:
That's what it compiles to anyway, @shaunandrews, and the way I see it code organization and readability are two of the main reasons why we even use SASS/SCSS. That's one of the things I always do as well: nest pseudo selectors to improve their readability for whoever sees this file next. See [this reference](https://sass-guidelin.es/#exceptions), but I guess it comes down to personal preference once again.

I'll clean up the code and merge!